func main() {
    var x int = 5
    
    // Erro: break fora de loop
    break
    
    // Erro: continue fora de loop
    continue
}
